vars/podman.groovy

```diff
@@ -0,0 +1,85 @@
+//Run podman remote build and push
+// Available parameters:
+//    arch:              string -- Build architecture
+//    tag:               string -- Image tag
+//    repo:              string -- Quay repository 
+def buildRemote(params = [:]) {
+    arch = params['arch']
+    repo = params['repo']
+    tag = params['tag'] 
+
+    withCredentials([
+        string(credentialsId: "fcos-${arch}-builder-host-string",
+               variable: 'REMOTEHOST'),
+        string(credentialsId: "fcos-${arch}-builder-uid-string",
+               variable: 'REMOTEUID'),
+        sshUserPrivateKey(credentialsId: "fcos-${arch}-builder-sshkey-key",
+                          usernameVariable: 'REMOTEUSER',
+                          keyFileVariable: 'CONTAINER_SSHKEY')
+    ]) {
```

Comment:
I don't think we should hardcode credential names in this shared library. Some options:
- remove `withCredentials` here: i.e. have the caller wrap calls to `buildRemote` with `withCredentials` itself
- make the credential names parameters: e.g. have a `host-credential` and a `uid-credential`.